packages/inspector/src/inspector.ts

```diff
@@ -115,18 +128,24 @@ export class InspectorPanel extends Panel
     this.source = null;
   }
 
-  private _content: Widget = Private.defaultContent();
+  /**
+   * Generate content widget from string
+   */
+  private static generateContentWidget(message: string): Widget {
```

Comment:
Our convention is that private methods start with underscore.

Reply:
updated with the fix